src/app/notes/notes-item/notes-item.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Note } from '../note.model';
+import { NotesService } from '../notes.service';
+
+@Component({
+  selector: 'app-notes-item',
+  templateUrl: './notes-item.component.html',
+  styleUrls: ['./notes-item.component.scss'],
+  providers: [NotesService]
```

Comment:
jezeli tak zrobisz prawdopodobnie kazda instacja komponentu bedzie miala swoja instacje serwisu z logika >> a to zle

jezeli w przyszlosci ten serwis ma byc bezstanowy to bez providers

Reply:
providers: [NotesService] jest poprawione w nastepnym komicie i jest dodane w tylko notes.module.ts, wiec pytanie czy to jest okey do zrobienia na module ?

Reply:
jezeli masz Injectable{providedIn: root  }to nie trzeba Ci juz w tablicy providers dodawac w zadnym modue